index.html

```diff
@@ -115,10 +115,9 @@ <h3>My Favourite Youtube Video</h3>
             <!--Found me!-->
 
             <picture>
-                <source type="image/svg" srcset="images/sc-logo.svg">
+                <source type="image/svg+xml" srcset="images/sc-logo.svg">
                 <source type="image/png" srcset="images/sc-logo.png">
-                <img src="images/sc-logo.png" srcset="images/sc-logo.svg 70w" alt="Sault College Logo"
-                    sizes="(max-width:105px) 105px, (max-width:140px) 140px">
+                <img src="images/sc-logo.png" srcset="images/sc-logo.png, images/sc-logo@1.5x.png 1.5x, images/sc-logo@2x.png 2x" alt="Sault College Logo" width="70">
             </picture>
```

Comment:
For resolution switching, no sizes attribute is required.  You only need a srcset, specifying screen densities for each source